src/testutils.jl

```diff
@@ -96,6 +98,23 @@ function pdtest_diag(C::AbstractPDMat, Cmat::Matrix, cmat_eq::Bool, verbose::Int
     end
 end
 
+function pdtest_cholesky(C::PDMat, Cmat::Matrix, cmat_eq::Bool, verbose::Int)
+    _pdt(verbose, "cholesky")
+    if cmat_eq
+        @test cholesky(C).U == cholesky(Cmat).U
+    else
+        @test cholesky(C).U ≈ cholesky(Cmat).U
+    end
+end
+
+function pdtest_cholesky(C::PDSparseMat, Cmat::Matrix, cmat_eq::Bool, verbose::Int)
+    _pdt(verbose, "cholesky")
+    if cmat_eq
+        @test SuiteSparse.CHOLMOD.Sparse(cholesky(C)) == SuiteSparse.CHOLMOD.Sparse(cholesky(Cmat))
```

Comment:
Travis is complaining about this:
```
Error During Test at /home/travis/build/JuliaStats/PDMats.jl/src/testutils.jl:113
  Test threw exception MethodError(SuiteSparse.CHOLMOD.Sparse, (Cholesky{Float64,Array{Float64,2}}([2.0 -1.0 -0.5; -2.0 2.0 -0.75; -1.0 -1.0 2.27761], 'U', 0),), 0xffffffffffffffff)
  Expression: (SuiteSparse.CHOLMOD).Sparse(cholesky(C)) == (SuiteSparse.CHOLMOD).Sparse(cholesky(Cmat))
  MethodError: no method matching SuiteSparse.CHOLMOD.Sparse(::Cholesky{Float64,Array{Float64,2}})
  Closest candidates are:
    SuiteSparse.CHOLMOD.Sparse(::Any) where T<:AbstractArray at abstractarray.jl:22
    SuiteSparse.CHOLMOD.Sparse(!Matched::Ptr{SuiteSparse.CHOLMOD.C_Sparse{Tv<:Union{Complex{Float64}, Float64}}}) where Tv<:Union{Complex{Float64}, Float64} at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/SuiteSparse/src/cholmod.jl:262
    SuiteSparse.CHOLMOD.Sparse(!Matched::Integer, !Matched::Integer, !Matched::Array{Int64,1}, !Matched::Array{Int64,1}, !Matched::Array{#s608,1} where #s608<:Union{Complex{Float64}, Float64}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/SuiteSparse/src/cholmod.jl:881
    ...
  Stacktrace:
   [1] SuiteSparse.CHOLMOD.Sparse(::Cholesky{Float64,Array{Float64,2}}) at ./abstractarray.jl:22
   [2] pdtest_cholesky(::PDSparseMat{Float64,SparseMatrixCSC{Float64,Int64}}, ::Array{Float64,2}, ::Bool, ::Int64) at /home/travis/build/JuliaStats/PDMats.jl/src/testutils.jl:113
```

Reply:
Yeah, I might just drop adding a cholesky function for PDSparseMat because I can't get master to build locally (issues with Arpack -> BinaryProvider -> libgfortran).

Reply:
You can use a nightly binary for Julia, since all of our official binaries package libgfortran. That should solve the ARPACK issues. Currently ARPACK doesn't seem to work with source builds of Julia.

Reply:
Yes. Please just use the official binary for package development.